metadata-ingestion/tests/integration/hive/test_hive.py

```diff
@@ -83,6 +83,56 @@ def test_hive_ingest(
 
     # Limitation - native data types for union does not show up as expected
 
+def presto_on_hive_config(events_file):
+
+    return {
+            "run_id": "hive-test",
+            "source": {
+                "type": "presto-on-hive",
+                "config": {
+                    "host_port": "localhost:5432",
+                    "database": "metastore",
+                    "database_alias": "hive",
+                    "username": "postgres",
+                    "scheme": "postgresql+psycopg2",
+                    "include_views": True,
+                    "include_tables": True,
+                    "schema_pattern": {"allow": ["^public"]},
+                    "mode": "hive",
+                },
+            },
+        "sink": {
+            "type": "file",
+            "config": FileSinkConfig(filename=str(events_file)).dict(),
+        },
+    }
+
+@freeze_time(FROZEN_TIME)
+@pytest.mark.integration_batch_1
+def test_presto_on_hive_ingest(
+    loaded_hive, pytestconfig, test_resources_dir, tmp_path, mock_time
+):
+    mce_out_file = "test_prest_on_hive_ingest.json"
+    events_file = tmp_path / mce_out_file
+
+    # Run the metadata ingestion pipeline.
+    pipeline = Pipeline.create(presto_on_hive_config(events_file))
+    pipeline.run()
+    pipeline.pretty_print_summary()
+    pipeline.raise_from_status(raise_warnings=True)
+
+    # Verify the output.
+    mce_helpers.check_golden_file(
+        pytestconfig,
+        output_path=events_file,
+        golden_path=test_resources_dir / "hive_mces_golden.json",
+        ignore_paths=[
+            # example: root[1]['proposedSnapshot']['com.linkedin.pegasus2avro.metadata.snapshot.DatasetSnapshot']['aspects'][0]['com.linkedin.pegasus2avro.dataset.DatasetProperties']['customProperties']['CreateTime:']
+            # example: root[2]['proposedSnapshot']['com.linkedin.pegasus2avro.metadata.snapshot.DatasetSnapshot']['aspects'][0]['com.linkedin.pegasus2avro.dataset.DatasetProperties']['customProperties']['Table Parameters: transient_lastDdlTime']
+            r"root\[\d+\]\['proposedSnapshot'\]\['com\.linkedin\.pegasus2avro\.metadata\.snapshot\.DatasetSnapshot'\]\['aspects'\]\[\d+\]\['com\.linkedin\.pegasus2avro\.dataset\.DatasetProperties'\]\['customProperties'\]\['.*Time.*'\]",
+            r"root\[6\]\['proposedSnapshot'\]\['com.linkedin.pegasus2avro.metadata.snapshot.DatasetSnapshot'\]\['aspects'\]\[\d+\]\['com.linkedin.pegasus2avro.schema.SchemaMetadata'\]\['fields'\]\[\d+\]\['nativeDataType'\]",
```

Comment:
replace `root\[6\]` with `root\[\d+\]` similar to the line above to make the test more robust against mcp reordering.